### DISCUSSION TITLE
Manually trigger emails

### DISCUSSION THREAD

**Discussion by grobmeier** [plausible/analytics#1898] [Self-Hosted Support] [answered]
Hello,

due to problems with email sending, I am wondering if I can trigger email sends with plausible manually to identify issues.
Anybody did that already?

Kind regards,
Christian

**Comment by Vigasaurus** [comment, marked as answer]
Either password resets or inviting users is likely your easiest on-demand email sending.

**Reply by grobmeier** [reply]
Thank you
